### PR TITLE
Create donatellotmnt.yml

### PR DESCRIPTION
**Pull request by superiorxman**
went last with the skin on this one

### CODE REVIEW

**Comment by superiorxman** [comment]
https://www.youtube.com/watch?v=wOm6lL3mly4
haste
0:11-0:18

**Comment by superiorxman** [comment]
https://www.youtube.com/watch?v=wOm6lL3mly4
jump
0:56-0:59

**Comment by superiorxman** [comment]
https://www.youtube.com/watch?v=WnoZnEupAyk
0:31 and up
Run

**Comment by superiorxman** [comment]
https://i0.wp.com/www.teenagemutantninjaturtles.com/wp-content/uploads/2012/10/Donatello.jpg
bow staff

File: CommandHelper/LocalPackages/commands/chars/donatellotmnt.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+jump: 2
+maxHearts: 20
+permissions:
+  superherocity:
+powers:
+- /effects
+- /items
+- /jump 0-4
+- /run 0-5
+- /haste 0-4
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/donatellotmnt.yml]: this haste, run, and jump doesnt match the values u put

Comment (superiorxman) [reply CommandHelper/LocalPackages/commands/chars/donatellotmnt.yml]: what would it be in your opinion then?

Comment (malonnnn) [reply CommandHelper/LocalPackages/commands/chars/donatellotmnt.yml]: it would be the values u used in the actual power, not what you listed in /powers

**Pull request merged**